concordium-android-sdk/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>com.concordium.sdk</groupId>
         <artifactId>concordium-sdk-base</artifactId>
-        <version>8.0.0</version>
+        <version>8.1.0</version>
```

Comment:
Is adding more enum variants not a breaking change? I would expect version `9.0.0` 🤔

Reply:
Yeah, you're right, we can't be sure that all the developers who use Java SDK have implemented graceful failing when encountering unknown enum entries. Although it will compile in Java, the runtime behavior could be unexpected.